SubsoniciousKit/SubsoniciousKit/Authentication/Server.swift

```diff
@@ -9,12 +9,12 @@
 import Foundation
 
 public struct Server {
-    public var address: String
+    public var baseURL: String
```

Comment:
having an url here could be better for you : better error handling at login time, better handling for network request

Reply:
I don't agree with that as mentioned in https://github.com/bilalb/Subsonicious/pull/5#discussion_r456769303